src/commands.ts

```diff
@@ -565,16 +563,7 @@ export class Commands {
 
   openSurvey = (surveyPrompt: SurveyPrompt) => {
     this.telemetry.sendEvent('openSurvey');
-    const extensionInfo = getExtensionInfo();
-
-    const query = querystring.stringify({
-      platform: encodeURIComponent(osName()),
-      vscodeVersion: encodeURIComponent(vscode.version),
-      extensionVersion: encodeURIComponent(extensionInfo.version),
-      machineId: encodeURIComponent(vscode.env.machineId),
-    });
-
```

Comment:
Possible improvements for future iterations of the form :
1. Would it be possible to add these as questions as well and pre-fill the csat form with this info if they came from here? 
2. Similarly, could we also pre-select the VSCode tool for them by passing it in as query param?

Reply:
i will create a followup ticket to address these